querybook/server/logic/data_element.py

```diff
@@ -26,6 +26,14 @@ def get_data_element_by_name(name: str, session=None):
     return DataElement.get(name=name, session=session)
 
 
+@with_session
+def get_query_metastore_has_data_elements(metastore_id, session=None):
```

Comment:
the function name (expects metastores) seems not match what it actually returns (boolean)